### ISSUE TITLE
Semantic attributes

### ISSUE THREAD

**Issue by cmussi** [w3c/epub-3-wg-charter#46]
I think we should work on a clear definition of semantic attributes in EPUBs (epub:type, DPub AIR, etc.).

For Dpub ARIA: we need to work with assistive technology developers to make users benefit from the work done by EPUB creators.

**Comment by iherman** [comment]
@cmussi I would agree but I think such a detailed list of technical features to plan for is something to be done by the WG (at its beginning) rather than listing them in the charter. The scope leaves the technical work open-ended and only emphasizes the safeguards for backward compatibility.

That being said: maybe it is worth making an informal list of issues that have been raised during the charter discussion (so that we would not forget them) and aim for the beginning of the WG's work to list them and see which ones should be handled by the WG and which by the CG? 

@wareid @dauwhe @mteixeira-wwn @rakutenjeff @TzviyaSiegman

**Comment by dauwhe** [comment]
> see which ones should be handled by the WG and which by the CG?

And which ones by other working groups. The semantics and accessibility of HTML is not just a concern for EPUB :)

**Comment by TzviyaSiegman** [comment]
I agree with @iherman. We should start preparing to do the work, but I don't think we want to be explicit in the charter. If we are too explicit it might limit what we work on in the future.

**Comment by iherman** [comment]
> That being said: maybe it is worth making an informal list of issues that have been raised during the charter discussion (so that we would not forget them) and aim for the beginning of the WG's work to list them and see which ones should be handled by the WG and which by the CG?

I have set an 'issue for the WG' label. I have added this label on a number of issues (including this one). Probably it is better _not_ to close such issues.
